Enterprise-CMCS/macpro-mdct-qmr
# reorder per David Ks comment

### Description
Updated from David's comment:
> One minor change and this is on me, can we make the order of the labels be Percentage of current smokers and tobacco users → Advising Smokers and Tobacco Users to Quit ->Discussing Cessation Medications → Discussing Cessation Strategies?


### Related ticket(s)


MDCT-2255

---
### How to test
Go to MSC-AD in 2023, answer first three questions, see in the Performance Measures that the order is as follows:
Percentage of Current Smokers and Tobacco Users
Advising Smokers and Tobacco Users to Quit
Discussing Cessation Medications
Discussing Cessation Strategies

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
